modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/alias/Alias.es.js

```diff
@@ -69,6 +69,21 @@ class Alias extends Component {
 		inputValue: '',
 	};
 
+	_handleBlur = () => {
```

Comment:
Can you add docs on what this function does? It could help in the future to easily see its purpose since it could not be easily clear to see seeing an `onChange` and then clearing the input.